### ISSUE TITLE
Biometric Lock for Windows App Similar to Android App

### ISSUE THREAD

**Issue by booberthecat** [standardnotes/forum#3758]
**Have you read the [Longevity Statement](https://standardnotes.org/longevity)?**
Yes, I did.

**Is your feature request related to a problem? Please describe.**
Yes, I believe my requests addresses a potential security issue in the Windows app whereby anyone who logs into Windows might be able to access the Standard Notes Windows app where a password has been saved.

**Describe the solution you'd like**
I would like you to add Biometric Lock security to the Windows app similar to what you have in the Android app whereby the user could configure the app settings to require biometrics either immediately or on quit to enter the app. I know that not every PC has a fingerprint reader but neither does every Android phone and availing ourselves of that technology on PC's that have it as so many other apps have done just makes good sense. I love the fact that on my phone I am forced to provide my fingerprint to access my Standard Notes data even if I am already using the phone. That extra level of security is reassuring and in my opinion missing in the PC app a necessary for an app that prides itself on security.

**Describe alternatives you've considered**
The only alternative I can think of is to erase any saved password in Windows and be forced to enter my credentials manually every time I wish to use the app. This alternative is cumbersome and inefficient in my opinion. 

**Additional context**
I came to Standard Notes after a long search for a secure, feature-rich app that met a long list of requirements for my particular use case -- it ended up being the clear winner for me. Only after I purchased it did I learn of its recent new connection to ProtonMail which I have used for several years. I finally found the notes tool I was looking for and I am very happy with it. This request is not a complaint but rather a request for a feature that I believe will benefit many users and make your app even more secure. Thank you for your consideration of my feature request. Feel free to reach out with any questions or needed clarifications.

**Comment by markriaboy** [comment]
I'm new to standardnotes, but I agree that a biometric lock might be a useful addition to the software. I also have the android app and find the feature quite helpful.

**Comment by olitala** [comment]
I agree with the importance of adding biometric lock in the desktop app as well.
